<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProposalDialog</class>
 <widget class="QDialog" name="ProposalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proposal Generator - Submit Proposal</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Please enter your proposal details below. NOTE: Successfully submitting the data will result in a transaction of 50 CHC being created and sent automatically.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QValidatedLineEdit" name="addressIn">
       <property name="toolTip">
        <string>The Chaincoin address to sign the message with</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addressBookButton">
       <property name="toolTip">
        <string>Choose previously used address</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/green/address-book</normaloff>:/icons/green/address-book</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteButton">
       <property name="toolTip">
        <string>Paste address from clipboard</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>Alt+P</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="urlIn">
     <property name="placeholderText">
      <string notr="true">Enter valid URL (https://chaincoin.org, www.example.com)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="proposalIn">
     <property name="documentTitle">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter description (256 characters max)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BitcoinAmountField" name="amountIn"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Payment date (earliest):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Payments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="toolTip">
        <string>Validate and submit data</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="toolTip">
        <string>Reset all sign message fields</string>
       </property>
       <property name="text">
        <string>Clear &amp;All</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>48</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="closeButton">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qt/qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>qt/bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
